Automattic/jetpack
# Boost: Re-fetch premium status on new connection

Fixes 55-gh-Automattic/boost-cloud

#### Changes proposed in this Pull Request:

* Re fetch all the optimizations and their status on the new Jetpack connection
* Once fetched run regenerate Cloud CSS via post connection tasks.

#### Jetpack product discussion


p1654148431975539-slack-C016BBAFHHS

**Note**: This PR is a quick fix for the issue. We need a correct way to reload all the configurations after the Jetpack connection is established. Also, the Cloud CSS front end module is not self-healing, ie if there is no request made before and the cloud CSS module is active it will show a waiting message indefinitely. This needs to be fixed properly within the cloud CSS module and the fix in this PR is temporary.
Ref: p1654241647693889-slack-C03J26VQ24V

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Purchase Jetpack Boost premium via Boost Dashboard
* Deactivate and re-activate Boost plugin
* Visit Jetpack Boost dashboard
* Ensure that the connection screen is shown
* Once connected, ensure that the Cloud CSS option is shown with a fresh Cloud CSS request
